text/0000-fabric-private-chaincode-lite.md

```diff
@@ -224,11 +215,14 @@ Arguably, it might even be better to put it _after_ the architecture as there ar
 
 - Organization do not have to trust any other organization (i.e., their users, admins and software such as peers) as far as confidentiality is concerned. That is, all of them could collude, modify any software (not only fabric but also operating systems, hypervisor) and still would not be able to extract private state from the chaincode or learn anything about the requests or responses of the victim organization (other than what the chaincode logic allows them to learn about it).
 
-- Organizations do have to trust a quorum of other organizations as defined by the lifecycle and chaincode endorsement policies as far as integrity of chaincode metadata is concerned. E.g., they have to rely on quorums of admins to only create and modify chaincodes as apropriate and they have to rely on quorum of peers to properly execute validation chaincode transactions.
+- Organizations do have to trust a quorum of other organizations as defined by the lifecycle and chaincode endorsement policies as far as integrity of chaincode metadata is concerned. E.g., they have to rely on quorums of admins to only create and modify chaincodes as appropriate and they have to rely on quorum of peers to properly execute validation chaincode transactions.
 
-- We also assume that users trust the peers of their own organization, e.g., when retrieving chaincode encryption keys. (This is primarily for simplicity. As for any Fabric chaincode, users outside of organizations could implement queries without trust in a single organization/peer by repeating queries withe multiple peers/organizations until enough identical responses are received to satisfy the endorsement policy, similar to transaction validation at peers before applying them to the ledger.)
+- We also assume that normally users trust the peers of their own organization, e.g., when retrieving chaincode encryption keys. (This is primarily for simplicity. As for any Fabric chaincode, users outside of organizations could implement queries without trust in a single organization/peer by repeating queries withe multiple peers/organizations until enough identical responses are received to satisfy the endorsement policy, similar to transaction validation at peers before applying them to the ledger.)
 
 - We do assume that a code running inside a TEE cannot be tampered with or its memory inspected. Similarly, we also require that remote attestation provided by a TEE are authentic and prove that only the code referenced in the attestation could have issued it.
+Therefore, all participants/organizations trust a TEE (in particular, the FPC Chaincode Enclave), which can provide such an attestation, regardless of at which peer/organization the TEE is hosted.
+
+- The TEE (FPC Chaincode Enclave) does not trust the hosting peer. That is, all data received via transaction invocations (e.g., the transaction proposal) or via state access operations (e.g., `get_state`) are considered untrusted.
```

Comment:
This is really also a correlary of above bullet and the second one (similar to the second paragraph of above bullet)?
```suggestion
  However, given above-mentioned trust assumptions on peers, a TEE (FPC Chaincode Enclave) cannot trust the hosting peer. Hence all data received via transaction invocations (e.g., the transaction proposal) or via state access operations (e.g., `get_state`) must be considered untrusted.
```

Reply:
updated